src/config.js

```diff
@@ -396,11 +396,16 @@ export default class Config {
     }
 
     if (process.platform === 'win32') {
-      if (this.plugnplayEnabled) {
-        this.reporter.warn(this.reporter.lang('plugnplayWindowsSupport'));
+      const cacheRootFolderDrive = path.parse(this._cacheRootFolder).root;
+      const lockfileFolderDrive = path.parse(this.lockfileFolder).root;
+
+      if (cacheRootFolderDrive !== lockfileFolderDrive) {
+        if (this.plugnplayEnabled) {
+          this.reporter.warn(this.reporter.lang('plugnplayWindowsSupport'));
+        }
```

Comment:
Now may be a good time to think on other platform specific path things that might make relative paths less portable from platform to platform. Off the top of my head there is also long paths _([namespaced paths](https://nodejs.org/api/path.html#path_path_tonamespacedpath_path))_ in Windows.

Reply:
I guess if the need arise we could just have a `plugnplay-use-absolute` settings that would use absolute paths for everything outside of `__dirname`. Since `getPackageInformation` already returns absolute paths no matter what, it should be backward compatible.